raiden/ui/cli.py

```diff
@@ -79,6 +89,54 @@ def toggle_trace_profiler(raiden):
         raiden.profiler.start()
 
 
+def wait_for_sync(blockchain_service, url, tolerance, sleep):
+    try:
+        oracle_block = quantity_decoder(requests.get(url).json()['result'])
```

Comment:
I have some other software which I query etherscan from, and in times of "high traffic" a request may need to be attempted 2 or 3 times with a small pullback. Can you maybe wrap this in such a retry logic?

Reply:
do you recall what are the errors like? requests / urllib3 have retries builtin as long as etherscan doesn't return an invalid response with valid http response headers

Reply:
it gave a response, but had an error inside. So status was error, server too busy or something like that.